start_pos: [0, 0]
goal_pos: [8, 8]
num_nodes: 100
k_nearest: 5
max_edge_dist: 2.0
x_range: [0, 10]
y_range: [0, 10]
obstacle_list: [[3, 3, 1], [6, 6, 1]]
